snippets/dockerfile.json

```diff
@@ -110,5 +110,14 @@
             "$0"
         ],
         "description": "ONBUILD"
+    },
+    "HEALTHCHECK": {
+        "prefix": "HEALTHCHECK",
+        "body": [
+            "HEALTHCHECK --interval=30s --timeout=30s --retries=3 CMD ${command}"
```

Comment:
Any thoughts on this @saurabsa? As I mentioned, I'm open to whatever works best.  The way I see it there are a few options.

1. list them with their default values as I did in my original commit
2. list them with variables as I mentioned in my last post (e.g. --interval=${interval})
3. leave them out completely.  As I mentioned, my only issue with this is that there would then be no reference to these options in the helper, however, it may be that that is the best option.

If there are any other options please let me know.

Thank you for your considerations.